uvloop/loop.pyx

```diff
@@ -3116,6 +3116,22 @@ cdef class Loop:
         await waiter
         return udp, protocol
 
+    def monitor_fs(self, path: str, callback, flags: int) -> asyncio.Handle:
+        p_bytes = path.encode('UTF-8')
+        cdef char* c_str_path = p_bytes
+        return self._monitor_fs(c_str_path, flags, callback)
+
+    cdef _monitor_fs(self, char* path, int flags, callback):
```

Comment:
Why is this `_monitor_fs()` an individual cdef method? If it was only for Cython performance, it would be the same to do it directly in `def monitor_fs()`.

Reply:
also changed per your comment